### ISSUE TITLE
venv: unicode and socket are not detected correctly

### ISSUE THREAD

**Issue by julian-r** [PyCQA/pylint#1174]
### Steps to reproduce
Windows Python 3.5.2/3.4.4 global install

1. create a virtualenv `python -m venv hello`
2. enter it `hello\scripts\activate`
3. install pylint `pip install pylint`
4. execute linter `pylint testscript.py`

`testscript.py`:
```
"foo module"

import unicodedata
import socket

print(unicodedata.normalize('NFC', 'Hello'))
print(socket.SHUT_RDWR)

```

### Current behavior
```
(hello) C:\Users\julian\tests>hello\Scripts\pylint testscript.py
No config file found, using default configuration
************* Module testscript
E:  6, 6: Module 'unicodedata' has no 'normalize' member (no-member)
E:  7, 6: Module 'socket' has no 'SHUT_RDWR' member (no-member)
```

### Expected behavior
No errors.

### pylint --version output
```
(hello) C:\Users\julian\tests>hello\Scripts\pylint --version
No config file found, using default configuration
pylint 1.6.4,
astroid 1.4.8
Python 3.4.4 (v3.4.4:737efcadf5a6, Dec 20 2015, 19:28:18) [MSC v.1600 32 bit (Intel)]
```

**Comment by PCManticore** [comment]
Not sure what might cause this. I tested on Linux and it works just fine, it might be that SHUT_RDWR is not available on Windows? I'll test with a Windows machine in the following days.

**Comment by PCManticore** [comment]
I managed to reproduce this on Windows, but only using the builtin **venv** module. With ``python -m virtualenv`` everything went fine. The problem seems to stem from the fact that **venv** does not add a ``sys.real_prefix`` attribute to point to the original installation, so that we can know where to look for builtin modules. This makes the inference engine to detect that there are these builtin modules, but cannot find their content, leading to this false positive you are seeing.

**Comment by PCManticore** [comment]
This should be fixed now in astroid's master. Can you give it a try with pylint==2.0 and astroid==1.5.0, both installed from their repositories? Let me know if they work for you.

**Comment by PCManticore** [comment]
The corresponding change is https://github.com/PyCQA/astroid/commit/60743e7e19f319af951948a3fcc2804f980293d3.

**Comment by julian-r** [comment]
Both the master version, right?

**Comment by julian-r** [comment]
Looks good, tested with the current git versions of pylint and astroid. Thanks!
